torchvision/prototype/transforms/_container.py

```diff
@@ -21,16 +19,21 @@ def forward(self, *inputs: Any) -> Any:
         return sample
 
 
-class RandomApply(_RandomApplyTransform):
-    def __init__(self, transform: Transform, *, p: float = 0.5) -> None:
-        super().__init__(p=p)
-        self.transform = transform
+class RandomApply(Compose):
+    def __init__(self, transforms: Sequence[Callable], *, p: float = 0.5) -> None:
```

Comment:
Is it BC that `p` is kwargs only now ?

Reply:
I have corrected this at https://github.com/pytorch/vision/pull/6512. I'm not sure what was the rational of splitting and cherrypicking my commits into multiple PRs but please make sure you won't miss any of the commits that fixes this.

Reply:
Well, maybe it is not the best approach. I did that as RandAug was still broken in your PR...

Reply:
Since we have merged #6512 before this one, it created a merge conflict.